app/controllers/users_controller.rb

```diff
@@ -0,0 +1,27 @@
+class UsersController < ApplicationController
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      session[:user_id] = @user.id
+      flash[:success] = "Logged in as #{@user.username}"
+      redirect_to dashboard_path
```

Comment:
It's nice that we can redirect right to the dashboard path without any params since we have the current_user helper